api/cloud-resources/v1beta1/awsnfsvolume_types.go

```diff
@@ -78,14 +78,18 @@ type AwsNfsVolumeStatus struct {
 	// +listType=map
 	// +listMapKey=type
 	Conditions []metav1.Condition `json:"conditions,omitempty" json:"conditions,omitempty"`
+
+	// +optional
+	// +kubebuilder:validation:Enum=Processing;Creating;Updating;Deleting;Ready;Error
+	State string `json:"state,omitempty"`
 }
 
-//+kubebuilder:object:root=true
-//+kubebuilder:subresource:status
+// +kubebuilder:object:root=true
+// +kubebuilder:subresource:status
+// +kubebuilder:printcolumn:name="Server",type="string",JSONPath=".status.server"
```

Comment:
Don't think we need to advertise the NFS server in print colum. @ngrkajac any thoughts?

Reply:
removed it now